testing/tests/cpp_nesting.py

```diff
@@ -119,10 +119,42 @@ def check_most_pages(self, uses_mainpage=True, main_refid="indexpage"):
         # TODO: make this a parameter if we add more than two tests...
         if uses_mainpage:
             f_name = "page_town_rock.hpp"
+            # Not sure why, but when it's a mainpage it's called "index" and when it's not it's called "overview"
```

Comment:
as said in the comment, depending on if it used a main page or not, the node is called 'index' or 'overview' respectively. This hacky solution seems to make the tests happy :)